README.md

```diff
@@ -36,15 +36,40 @@ Run `yarn lint` to run the linter, or run `yarn lint:fix` to run the linter and
 
 ### Release & Publishing
 
-The project follows the same release process as the other libraries in the MetaMask organization:
-
-1. Create a release branch
-   - For a typical release, this would be based on `main`
-   - To update an older maintained major version, base the release branch on the major version branch (e.g. `1.x`)
-2. Update the changelog
-3. Update version in package.json file (e.g. `yarn version --minor --no-git-tag-version`)
-4. Create a pull request targeting the base branch (e.g. `main` or `1.x`)
-5. Code review and QA
-6. Once approved, the PR is squashed & merged
-7. The commit on the base branch is tagged
-8. The tag can be published as needed
+The project follows the same release process as the other libraries in the MetaMask organization. The GitHub Actions [`action-create-release-pr`](https://github.com/MetaMask/action-create-release-pr) and [`action-publish-release`](https://github.com/MetaMask/action-publish-release) are used to automate the release process; see those repositories for more information about how they work.
+
+1. Choose a release version.
+
+- We follow SemVer, so the set of changes being released dictates what the release version will be. Analyze the changes to see whether they include any breaking changes, new features, or deprecations. See [the SemVer specification](https://semver.org/) for more information.
```

Comment:
```suggestion
- Follows Semantic Versioning  -- the set of changes being released dictates what the release version will be. Analyze the changes to see whether they include any breaking changes, new features, or deprecations. See [the SemVer specification](https://semver.org/) for more information.
```

Reply:
Hah, I was just rewriting this too. Here's my attempt:
```suggestion
- The release version should be chosen according to SemVer. Analyze the changes to see whether they include any breaking changes, new features, or deprecations, then choose the appropriate SemVer version. See [the SemVer specification](https://semver.org/) for more information.
```

I wanted to avoid the sentence fragment part at the start. How does this sound?